src/app/banking/vouchers/redeem-voucher.component.html

```diff
@@ -0,0 +1,66 @@
+<page-layout [ready]="data$ | async" [ngSwitch]="step$ | async">
+  <page-content *ngSwitchCase="'form'" [heading]="i18n.voucher.redeem.title"
+    [mobileHeading]="i18n.voucher.redeem.mobileTitle">
+
+    <input-field type="text" [label]="i18n.voucher.token" [mask]="mask"
+      [formControl]="token" [placeholder]="mask"></input-field>
+    <actions>
+      <button class="btn btn-primary" [disabled]="requesting$ | async"
+        (click)="submit()">{{ i18n.general.submit }}</button>
+    </actions>
+  </page-content>
+  <page-content *ngSwitchCase="'confirm'"
+    [heading]="i18n.voucher.redeem.title"
+    [mobileHeading]="i18n.voucher.redeem.mobileTitle">
+    <ng-container *ngIf="(dataForRedeem$ | async); else loading">
+
+      <notification ngClass="mb-3" type="warning">
+        {{ i18n.voucher.redeem.confirm }}
+      </notification>
+
+      <div class="d-flex flex-grow-1 flex-column">
+
+        <div
+          [ngClass]="(layout.ltsm$ | async)? '' : 'd-flex flex-row-reverse justify-content-between'">
+          <div *ngIf="dataForRedeem.type.image"
+            [ngClass]="dataForRedeem.type.voucherDescription ? 'image d-flex justify-content-center mt-2 mb-4' :  'image d-flex flex-grow-1 justify-content-center mt-2 mb-4'">
```

Comment:
Here, you can use class="static1 static2" [ngClass]="{'dynamic1': condition1, 'dynamic2': condition2}".
So, for example:
```suggestion
            class="image d-flex justify-content-center mt-2 mb-4"
            [ngClass]="{'flex-grow-1':  dataForRedeem.type.voucherDescription}">
```